### ISSUE TITLE
Issues in Vehicle building ; Vehicles won't drive unless you >add< more weight to it.

### ISSUE THREAD

**Issue by Kadian** [CleverRaven/Cataclysm-DDA#20039]
Apparently a bug that persists through several versions by now.

The  short version: Vehicles don't want to drive, apparently combined with an error message that the vehicle is too heavy for its engine. Adding a frame and/or any other part to the rear end of the vehicle, therefore adding to the weight, makes it drivable again. Remove said rear part and it stops working again.

That's at least how I understood it. For a more indepth read, please check this thread in our forum:
http://smf.cataclysmdda.com/index.php?topic=13983.15

If there is already an opened issue regarding this, sorry.

**Comment by keyspace** [comment]
> is too heavy for its engine

Only occurence is in `vehicle::thrust()`.

Forum link has savefile on first page to tinyupload, which is tagged virus by my system.

**Comment by keyspace** [comment]
Anyway, a save file would be nice (ZIP, attached here on GH). Preferably with a non-working vehicle and all tools/parts needed to make modifications.

**Comment by kevingranade** [comment]
Hmm, it might be that it's actually too imbalanced and the error message is
too generic, will need to try and test that.

**Comment by ituluwituluwzev** [comment]
Possibly duplicates issue #19201.

Version 6831.

My hippie van broke the right rear tile before it is parked. Then "the Hippie Van is too heavy for its engine(s)!", when I try to start. I install a wheel (17'', matching the other three) to replace the broken one, it is still "too heavy".

**Comment by Shinino** [comment]
I just ran into this -- I had a mobile meth lab that I stripped down, replacing a lot of the frame with HD frame, military composite, et al -- but I didn't have enough so left like 2/3 of the vehicle an outline.  It was 14000 pounds and I was getting that message.  I stripped it all the way down to an exterior (10000 pounds) and same thing.  Started re-adding all the HD frame pieces and internals -- at 16000 pounds I still get the message, but now it'll move.  That seems to line up with it not wanting to move because the weight is -not enough-.

After talking with people in Discord, its seems that location of engines compared to the location of the wheels may be the issue.  I moved my engines within the rectangle created by my wheels and now have no issues (at just under 17k pounds)
